app/main.py

```diff
@@ -1,8 +1,33 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int, average_rating: float, count_of_ratings: int):
```

Comment:
Place all arguments in the `__init__` on a new line including `self` for better code style. Make the same change on line 2 as well.